addons/mail/static/src/models/suggestable.js

```diff
@@ -0,0 +1,31 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { many, one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'Suggestable',
```

Comment:
name not really good out of context, if you have a better idea

Reply:
Name's good, though will add prefix "Composer" just to make sure it's clearer what's the context of these suggestable items.